.gitignore

```diff
@@ -2,3 +2,5 @@
 typings/
 node_modules
 admin.txt
+controllers/listener-quiz.js
```

Comment:
I don't think its a good idea to add these things to the gitignore, unless we also include documentation telling people to modify the gitignore for their own needs.

If a user down the road creates a `listener-quiz` with pushkin-cli, it would be gitignored and could be confusing.